### PR TITLE
CLDR-16704 markdown in infopanel

### PR DESCRIPTION
**Pull request by srl295**
- use marked (same as in tr-archiver)
- update html to use links in new window
- update 1 path into markdown as a test (exemplars)

CLDR-16704

- [ ] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
image
### 

image


image

**Comment by srl295** [comment]
image
`<a target="_blank" …` on all links

**Comment by srl295** [comment]
This could be a good candidate to put on staging if anyone wants to take a closer look at the generated HTML.

I also wonder if we need a new structure for the markdown source, it may not work in Java as easily

**Comment by macchiati** [comment]
Up until v40, it was data-driven, in a text file. Placeholders.txt still is. I think the change was to have redirection for the paths.

We should consider reverting to data-driven, but use a bit more structure. Eg

v40
```^//ldml/units/unitLength\[@type="([^"]*)"]/unit\[@type="area-acre"]/displayName ; Display name ({1} form) for “area-acre”, referring specifically to an English acre. Note: before translating, be sure to read http://cldr.org/translation/units-1/units.
```
v43
```
 + "^//ldml/units/unitLength\\[@type=\"([^\"]*)\"]/unit\\[@type=\"area-acre\"]/displayName"
+ RegexLookup.SEPARATOR
+ "Display name ({1} form) for “area-acre”, referring specifically to an English acre. Note: before translating, be sure to read "
+ CLDRURLS.UNITS_HELP
+ ".\n"
```

=> ?? v44
PathDescriptions.txt

```
path=^//ldml/units/unitLength\[@type="([^"]*)"]/unit\[@type="area-acre"]/displayName
message=Display name ({1} form) for “area-acre”, referring specifically to an English acre.
link=UNITS_HELP
```

And a separate file:

HelpLinks.txt

```
UNITS_HELP=http://cldr.org/translation/units-1/units
```

Possibly a json format. (Not suggesting XML; that's overkill)

**Comment by macchiati** [comment]
> This could be a good candidate to put on staging if anyone wants to take a closer look at the generated HTML.

Yes, please do

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java [hunk 1]
```diff
@@ -143,9 +143,9 @@ public class PathDescription {
                     + ".\n"
                     + "^//ldml/characters/exemplarCharacters$"
                     + RegexLookup.SEPARATOR
-                    + "Defines the set of characters used in your language. You may not edit or vote on this item at this time. Before filing any tickets to request changes, be sure to read "
-                    + CLDRURLS.EXEMPLAR_CHARACTERS
-                    + ".\n"
+                    + String.format(
```
Comment (macchiati) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/PathDescription.java:146]: I don't think this was necessary (filed https://unicode-org.atlassian.net/browse/CLDR-16755) but also not a blocker.

File: tools/cldr-apps/js/src/esm/cldrDeferHelp.mjs [hunk 2]
```diff
@@ -1,6 +1,7 @@
 /*
  * cldrDeferHelp: encapsulate code related to showing language descriptions in the Info Panel
  */
+import { marked } from "./cldrMarked.mjs";
```
Comment (btangmu) [inline tools/cldr-apps/js/src/esm/cldrDeferHelp.mjs:4]: @srl295 this is failing for me locally

ERROR in ./src/esm/cldrDeferHelp.mjs 13:15-21
export 'marked' (imported as 'marked') was not found in './cldrMarked.mjs' (module has no exports)

Comment (btangmu) [reply tools/cldr-apps/js/src/esm/cldrDeferHelp.mjs:4]: never mind, it's OK with clean rebuild

**Pull request merged**